src/cms/adobe-stock-manage.md

```diff
@@ -0,0 +1,210 @@
+
+1. Click <span class="btn">Filters</span>.
+
+    There are several filters available to refine your search results:
+
+    |Filter|Description|
+    |---|---|
+    |Subcategory|Filter for images that are **Photos** or **Illustrations**|
+    |Orientation|Filter for images by size, shape and aspect|
+    |Color|Use a color pallette to filter for images by color|
+    |Price|Filter for images based on their cost|
+    |Safe search|Enable or disable Safe search|
+    |Isolated Assets|Show only Isolated Assets, which have subjects appear alone on a solid background|
+
+    ![Adobe Stock search filters]({% link images/images/adobe-stock/filters.png %}){: .zoom}
+    _Adobe Stock Search Filters_
+
+1. Click **Apply Filters**.
+
+    The search result grid is updated with your refined search.
+
+## View Image Details
+
+Each image has details available for viewing. Additional image-specific actions, such as [saving image previews][save-preview] or [saving (and optionally licensing) images][save-licensed], are available via this detailed view.
+
+1. [Access the Adobe Stock Search grid][access-search].
+
+1. Click an image in the search results.
+
+    Further image details are displayed, such as:
+    - A larger version of the image
+    - Image metadata, such as **Dimensions**, **File type**, **Category**, **File#**, and **Keywords**
+    - Related images, such as images from the same **series** or **model**
+    - Action buttons, such as [**Save Preview**][save-preview] and [**Save (and optionally license) Image**][save-licensed]
+
+    ![Adobe Stock image details]({% link images/images/adobe-stock/image-details.png %}){: .zoom}
+    _Adobe Stock image details_
+
+## Log In to Your Adobe Account
+
+To gain complete access to an image and eliminate the Adobe Stock watermark, you must [sign in with an Adobe account][adobe-signin] and purchase credits to
+license rights to use an image.
+
+1. [Access the Adobe Stock Search grid][access-search].
+
+1. Click **Sign In** at the top-right.
+
+    A new browser window guides you through the [Adobe sign-in process][adobe-signin].
+
+    After completing the sign-in process, the [licensed state][licensed-state] of images is displayed in search results as an additional label.
+
+    ![Adobe sign in]({% link images/images/adobe-stock/ims-login.png %}){: .zoom}
+    _Adobe account sign in_
+
+### View the Licensed State of Search Results
+
+[Log in to your Adobe account][log-in-to-adobe-account].
+
+All licensed images associated to your Adobe account will have an additional label displayed on them, making it clear which images you have licensed.
+
+![Adobe Stock search results with licensed images]({% link images/images/adobe-stock/licensed-images.png %}){: .zoom}
+_Search Results with Licensed Images_
+
+### Save Images to the Media Storage
+
+Images searched for via the Adobe Stock Integration can be saved to the Magento [Media Storage][media-storage] for easy re-use across your Magento
+store.
```

Comment:
soft return?

Reply:
doesn't affect output, but we will fix later